### PR TITLE
Fix t.anal/arm/vars

### PR DESCRIPTION
**Pull request by MaskRay**
Output changes b/c of `shdr -> phdr`

### CODE REVIEW

**Comment by radare** [comment]
Why do you remove comments?

**Comment by Maijin** [comment]
Do not mark as broken, fix !, those are important tests!

File: t.anal/arm/vars [hunk 1]
```diff
@@ -106,6 +105,7 @@ run_test
 
 NAME="ELF ARM: af and aav"
 FILE=../../bins/elf/arm1.bin
+BROKEN=1
```
Comment (Maijin) [inline t.anal/arm/vars]: No, don't mark this as broken, this is an important test!

File: t.anal/arm/vars [hunk 2]
```diff
@@ -6,21 +6,20 @@ NAME="ELF ARM: function arg"
...
 e asm.calls=false
+e asm.comments=false
 afr@main
 afva@sym.call
 pdf@sym.call
 '
 EXPECT='            ;-- $a:
 / (fcn) sym.call 44
 |           ; var int local_8h @ fp-0x8
-|              ; CALL XREF from 0x00010468 (sym.main)
```
Comment (Maijin) [inline t.anal/arm/vars]: Why???

**Comment by MaskRay** [comment]
For [XX]  1  vars: ELF ARM: function arg 

Now it has a different display:
```diff
-|           0x0001042c      08000be5       str r0, [fp - local_8h]
+|           0x0001042c      08000be5       str r0, [fp - local_8h]     ; 8
```

`; 8` comes from `core/disasm.c:ds_print_ptr`  `	ut64 p = ds->analop.ptr;`

**Comment by radare** [comment]
lgtm to be shown, and probably @kazarmi will take care about that soon or late

File: t.anal/arm/vars [hunk 3]
```diff
@@ -95,7 +66,6 @@ aav
 pd 3 @ 0x00010328
 '
 EXPECT='            ;-- $d:
-               ; UNKNOWN XREF from 0x00010988 (section_end..eh_frame + 1160)
```
Comment (MaskRay) [inline t.anal/arm/vars]: d366ca79c619ef9f40c7897773c3697a9bd3ffa7 #8329 SHDR -> PHDR
the referer is not mapped.

File: t.anal/arm/vars [hunk 4]
```diff
@@ -75,9 +46,9 @@ EXPECT='            ;-- main:
 |           0x00010450      00482de9       push {fp, lr}
 |           0x00010454      04b08de2       add fp, sp, 4
 |           0x00010458      08d04de2       sub sp, sp, 8
-|           0x0001045c      08000be5       str r0, [fp - local_8h]
-|           0x00010460      0c100be5       str r1, [fp - local_ch]     ; 0xc
-|           0x00010464      08001be5       ldr r0, [fp - local_8h]
+|           0x0001045c      08000be5       str r0, [fp - local_8h]     ; 8
+|           0x00010460      0c100be5       str r1, [fp - local_ch]     ; 0xc ; 12
```
Comment (MaskRay) [inline t.anal/arm/vars]: It is supposed to display both 0xc and 12?

File: t.anal/arm/vars [hunk 5]
```diff
@@ -30,34 +29,6 @@ EXPECT='            ;-- $a:
 '
 run_test
 
-NAME="ELF ARM: function arg .. without io.sectonly, wtf siol"
```
Comment (MaskRay) [inline t.anal/arm/vars]: `io.sectonly` is removed, so this test is no longer needed.

**Comment by MaskRay** [comment]
brought back.

I don't know what to do with `vars: ELF ARM: af and aav`

**Comment by radare** [comment]
well, its a bit dirty, but its not incorrect. update the test and let it go

**Comment by radare** [comment]
agree, can be removed

File: t.anal/arm/vars (hunk 5 above)
Comment (Maijin) [reply t.anal/arm/vars]: Ok for this one to disappear

**Comment by Maijin** [comment]
vars: ELF ARM: af and aav

This one needs to be fixed

**Comment by alvarofe** [comment]
Let me take a look to that one, likely today or tomorrow

**Comment by alvarofe** [comment]
Fixed vars: ELF ARM: af and aav in master

File: t.anal/arm/vars (hunk 3 above)
Comment (alvarofe) [reply t.anal/arm/vars]: Was wrong cause at dir 0x00010988

0x00010988      bl sym._int_malloc

**Pull request merged**